--- .gitmodules
+++ .gitmodules
@@ -1,32 +1,32 @@
 [submodule "binutils"]
	 path = binutils
-	url = https://sourceware.org/git/binutils-gdb.git
+	url = https://github.com/bminor/binutils-gdb.git
	 branch = binutils-2_42-branch
 [submodule "gcc"]
	 path = gcc
-	url = https://gcc.gnu.org/git/gcc.git
+	url = https://github.com/gcc-mirror/gcc.git
	 branch = releases/gcc-13
 [submodule "glibc"]
	 path = glibc
-	url = https://sourceware.org/git/glibc.git
+	url = https://github.com/bminor/glibc.git
 [submodule "dejagnu"]
	 path = dejagnu
	 url = https://git.savannah.gnu.org/git/dejagnu.git
	 branch = master
 [submodule "newlib"]
	 path = newlib
-	url = https://sourceware.org/git/newlib-cygwin.git
+	url = https://github.com/mirror/newlib-cygwin.git
	 branch = master
 [submodule "gdb"]
	 path = gdb
-	url = https://sourceware.org/git/binutils-gdb.git
+	url = https://github.com/bminor/binutils-gdb.git
	 branch = gdb-14-branch
 [submodule "qemu"]
	 path = qemu
-	url = https://gitlab.com/qemu-project/qemu.git
+	url = https://github.com/qemu/qemu.git
 [submodule "musl"]
	 path = musl
-	url = https://git.musl-libc.org/git/musl
+	url = https://github.com/bminor/musl.git
	 branch = master
 [submodule "spike"]
	 path = spike
